File: Machine Learning/Matlab code/threshold_matrix.m
function y = threshold_matrix (x)

[row,col] = size(x);

for i=1:row
    for j=1:col
        if x(i,j) < 0
            x(i,j) = 0;
        else
            x(i,j) = 1;
        end
    end
end

y = x;

end

File: project/nnet/threshold_matrix.py
'''
threshold of array
'''


import numpy as np


def threshold_matrix(array):
	row, col = len(array), len(array[0])

	for i in range(row):
		for j in range(col):
			if array[i][j] < 0:
				array[i][j] = 0
			else:
				array[i][j] = 1
	y=array
	return y
